### DISCUSSION TITLE
How can run a nushell script in windows PowerShell?

### DISCUSSION THREAD

**Discussion by danli349** [nushell/nushell#11480] [General]
Hello :
How can I run a nushell script in Windows PowerShell? 

```bash
'C:\Program Files\nu\bin\nu.exe' D:\books\nushell\myscript.nu
At line:1 char:34
+ 'C:\Program Files\nu\bin\nu.exe' D:\books\nushell\myscript.nu
+                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Unexpected token 'D:\books\nushell\myscript.nu' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken
```
Thanks

**Comment by fdncred** [comment]
It's hard to tell without seeing your script but you can run scripts from powershell.
```
D:\nushell main
❯ cat .\test.nu
echo "Hello from Nushell"

D:\nushell main
❯ nu .\test.nu
Hello from Nushell

D:\nushell main
❯ pwsh --version
PowerShell 7.4.0
```

**Comment by danli349** [comment]
@fdncred 

I want to open these pdf files when my computer start up, so I in the "C:\Users\lidan\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup" folder, I added a "powershell.cmd" file, which contains 
```bash
PowerShell -Command "Set-ExecutionPolicy Unrestricted" >> "%TEMP%\StartupLog.txt" 2>&1
PowerShell D:\python\powershell_script3.ps1 >> "%TEMP%\StartupLog.txt" 2>&1
```
to run the "powershell_script3.ps1" when the computer starts up.

The "powershell_script3.ps1" file contains:
```bash
nu D:\books\nushell\myscript.nu
```

The nushell script "myscript.nu" is very simple, It just opens 2 pdf files:
```bash
cd `D:\books\math\Probability\0. Monte Carlo Markov Chain\0. Introducing Monte Carlo Methods with R`
open files | lines | par-each { |file| `C:\Program Files (x86)\Adobe\Acrobat DC\Acrobat\Acrobat.exe` [$file]}
```

I can double-click "powershell.cmd" file to open these pdf files, but these files can't open when the computer starts up.

Thanks

**Reply by fdncred** [reply]
One approach is to just make your nushell script
```nushell
start first.pdf
start second.pdf
```
This assumes that pdf is associated with acrobat on your computer. I'm not sure that will work either but it's worth a try.

**Comment by danli349** [comment]
@fdncred 
The old nu script can run successfully, 
But Powershell can't run the `myscript.nu` when start up.
```bash
nu D:\books\nushell\myscript.nu
```
